jobSeeker

Shivambu17 opened the pull request.

Shivambu17 committed:
jobSeeker

ICTPass2002kgao commented:
@Shivambu17 i saw no change which brings error but you can go through ona's changes since she did some updates then you can help her in intergrating other functionalities
here are some mistakes that ona have fixed so you can add your functionality on ona's Branch
![Screenshot (89)](https://github.com/user-attachments/assets/9d17df4f-61ab-41a5-85da-8e166c98709f)
![Screenshot (90)](https://github.com/user-attachments/assets/40be3ec4-1102-44b1-8bd3-07db2b3911e8)

Shivambu17 commented:
check on personal details i made some update like where there is other option i added a textbox when you click it also when  on provinces i added an options to choose instead of typing also on date of birth  i changed the format to be date whe you choose instead of typing
